src/postamble.js

```diff
@@ -246,8 +246,12 @@ function callMain(args) {
       return;
     } else {
       var toLog = e;
+      // add the stacktrace, unless its already included in the exception
+      // message itself (jsStackTrace does this).
       if (e && typeof e === 'object' && e.stack) {
-        toLog = [e, e.stack];
+        if (!e.stack.includes('jsStackTrace')) {
```

Comment:
this is a little unfortunate in increasing code size and also not working in legacy browsers. can we just avoid adding the stack at all here? as you say jsStackTrace adds it manually, that might be good enough.

Reply:
What do you mean not working in legacy browsers?

IIRC if we don't at it here we don't see the stacktrace for all normal exceptions that don't come through abort.  So any normal exceptions from JS would not include the stack trace at all.

I think the problem is that there are two types of exception that come through here:

1. abort() generated exceptions that already contains a stack trace in the message itself
3. All other exceptions... things like syntax errors even.

Reply:
Old-enough browsers don't support `.includes`.

I think it might be ok to not show a stack trace here, since the browser devtools will do so anyhow. Maybe the only risk is on Node. So one way to avoid the code size cost might be to ifdef on that.

Reply:
(but maybe even on node this isn't needed - that would be optimal)

Reply:
Without this code here, node will not show normal exception stacktraces.   (It will show abort stack traces since here we use  jsStackTrace to inject the stacktrace into the message).

Reply:
I see.

It seems unfortunate to increase code size and complexity here just for `jsStackTrace`. That is, it's annoying that `jsStackTrace` needs to leak to a very different part of the codebase.

Where is the call to `jsStackTrace` that is relevant here?